Gemfile

```
@@ -1,4 +1,5 @@
-# A sample Gemfile
 source "https://rubygems.org"
 
 gemspec
+
+gem 'pry'
```

Could this be put in an optional group?

```suggestion
group :development, optional: true do
  gem 'pry'
end
```

---

Added in https://github.com/DataDog/datadog-ruby_core_source/pull/8/commits/c3e6c4c9a3b77eeeae58aed318fc96ccfd4a97cd